src/setupTests.js

```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
+// learn more: https://github.com/testing-library/jest-dom
+import '@testing-library/jest-dom';
```

Comment:
You did well with your project, kindly npote that your design is different from the framework given. Although styling isn't meant to be elaborately followed but you are meant to follow the given framework to ensure consistency in design 😄 

Your design :
![image](https://github.com/mohamedgamaleldin999999/The-bookstore/assets/20537712/844f567a-4f21-441a-8ff1-d3157f69fa9d)

design given :
![image](https://github.com/mohamedgamaleldin999999/The-bookstore/assets/20537712/113d4b97-5060-4e96-8532-81a55ea4efc6)

NB : Your design dosn't need to be complete at this stage but it should follow the framework

Reply:
Hello, @usorfaitheloho. Thank you for your review. I understand your point. however, the deadline is closing in and it will be a risk to modify the styling now. It clearly states that styling is not required at this phase. so, I am hoping if you would be so kind to approve as is and I will have to style in future blocks when styling is required. Otherwise, please let me know now so I can delete all CSS files altogether and keep with no styling. Thanks.

Reply:
Hello @mohamedgamaleldin999999  although styling is not a major requirements but you are meant to follow the framework, kindly remove the place I highlighted and leave just the displayed book items

Reply:
ok I removed the percentage and chapter display. Can you check please? @usorfaitheloho . Thanks again for your help.